public/pages/CreateTrigger/containers/ConfigureTriggers/ConfigureTriggers.js

```diff
@@ -131,7 +136,7 @@ class ConfigureTriggers extends React.Component {
       case SEARCH_TYPE.QUERY:
       case SEARCH_TYPE.GRAPH:
         const searchRequest = buildRequest(formikValues);
-        _.set(monitorToExecute, 'inputs[0].search', searchRequest);
+        _.set(monitorToExecute, 'inputs[0]', searchRequest);
```

Comment:
This change has been checked in as part of [this PR](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/382), so just ensure there are no conflicts

Reply:
Rebasing the recent changes to `main` onto this dev branch has removed this duplicate change from this PR.